.github/workflows/diff_report.yml

```diff
@@ -85,16 +86,19 @@ jobs:
   make_report:
     runs-on: ubuntu-latest
     needs: parse_body
-    if: needs.parse_body.outputs.projects_link != ''
-          && (needs.parse_body.outputs.config_link != ''
-            || needs.parse_body.outputs.new_module_config_link != '')
+    if: needs.parse_body.outputs.config_link != ''
+            || needs.parse_body.outputs.new_module_config_link != ''
     outputs:
       message: ${{ steps.out.outputs.message}}
     steps: 
        
      - name: Download files
        run: |
-        wget -q "${{needs.parse_body.outputs.projects_link}}" -O project.properties
+        if [ -n "${{needs.parse_body.outputs.projects_link}}" ]; then
```

Comment:
If this is a bash script, then it can be changed to 
```bash
LINK="${{needs.parse_body.outputs.projects_link}}"
LINK="${LINK:-$DEFAULT_PROJECTS_LINK}"
wget -q "$LINK" -O project.properties
```

Reply:
it looks more cryptic for me. Are you sure it is better?

Reply:
But very functional style :) , the same as we have in java when we substitute IF with chained calls.

I do not mind on both variants.

Reply:
done.
I tested new script - looks like it is working fine

Reply:
We need to understand that art in GitHub action code will cost manual re testing, that takes time.
In java we do not have this pain

Reply:
There is thing like https://github.com/nektos/act#overview--- to let it run locally